service/controller/resource/tccpn/template/template_main_iam_policies.go

```diff
@@ -383,17 +383,17 @@ const TemplateMainIAMPolicies = `
               Federated: "arn:{{ .IAMPolicies.RegionARN }}:iam::{{ .IAMPolicies.AccountID }}:oidc-provider/{{ .IAMPolicies.CloudfrontDomain }}"
             Action: "sts:AssumeRoleWithWebIdentity"
             Condition:
-              StringEquals:
-                "{{ .IAMPolicies.CloudfrontDomain }}:sub": "system:serviceaccount:kube-system:external-dns"
+              StringLike:
+                "{{ .IAMPolicies.CloudfrontDomain }}:sub": "system:serviceaccount:kube-system:*"
```

Comment:
Do we want to also allow different namespaces? If so we also must replace kube-system with *

Reply:
I think leaving this in kube-system is the right approach... as it offer a bit of additional security.

@T-Kukawka also worth mentionting for release 19 upgrades

Will try to find out if customers install in other NS

Reply:
@whites11 as the SA depend on the app name, we can't do that :(

we can probably get away with external-dns* as the SA name on any namespace

Reply:
![Screenshot 2023-04-18 at 11 33 43](https://user-images.githubusercontent.com/1133954/232735886-409909e0-af90-483c-a1a5-920a9ac3006f.png)
customer example